### DISCUSSION TITLE
Build Vite Frontend with Bun

### DISCUSSION THREAD

**Discussion by ninoslat1** [oven-sh/bun#13812] [Show and tell]
Currently, i tried to build my Vite React apps and follow the documentation in here [https://bun.sh/guides/ecosystem/vite](url). However i encountered an error, where I can't serve the static HTML after running `bunx --bun vite build`. This is generated HTML from that command.

![image](https://github.com/user-attachments/assets/2b34b46d-1175-45d3-a68f-e7aedf8b16c3)

After trying to serve it with Live Preview, I got an error `Failed to load resource: the server responded with a status of 404`, this error happens because we need to add `.` before the `/assets`. So we need to configure the vite.config.ts with adding `base: './` like this

```javascript
export default defineConfig({
  plugins: [react()],
  base: './',
})
```

After adding base path from the config and rebuilding the apps, we got this HTML, and when I used Live Preview again. The source was successfully loaded

![image](https://github.com/user-attachments/assets/82f96e7f-327b-410d-b02a-1b317c7af625)

**Comment by Nanome203** [comment]
I think this might help: https://www.reddit.com/r/reactjs/comments/161sj18/vite_build_not_working/?rdt=50079

Seems like it only works when you deploy it to a web server

**Reply by ninoslat1** [reply]
I see, thank you for your reply
